src/tests/base_test.py

```diff
@@ -153,6 +207,42 @@ def new_cluster_configuration(self, request: FixtureRequest) -> ClusterConfig:
 
         return config
 
+    @pytest.fixture
+    def new_day2_cluster_configuration(
+        self, request: FixtureRequest, cluster: Cluster, triggers_enabled, triggers
+    ) -> Day2ClusterConfig:
+        """
+        Creates new day2 cluster configuration object.
+        Override this fixture in your test class to provide a custom cluster configuration. (See TestInstall)
+        :rtype: new day2 cluster configuration object
+        """
+        config = Day2ClusterConfig()
+        self.update_parameterized(request, config)
+
+        if triggers_enabled:
```

Comment:
I had to move the trigger for the day2 configuration here, otherwise it creates a dependency cycle between the day1 and day2 clusters